compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/input/key/KeyEvent.web.kt

```diff
@@ -41,6 +41,18 @@ private fun KeyboardEvent.toInputModifiers(): PointerKeyboardModifiers {
     )
 }
 
+/**
+ * The original raw native KeyboardEvent event
+ *
+ * Null if:
+ * - the native event is sent by another framework (when Compose UI is embed into it)
+ * - there is no native event (in tests, for example, or when Compose sends a synthetic event)
+ *
+ */
+val KeyEvent.webEventOrNull: KeyboardEvent?
```

Comment:
It should be in platform-specific package, not in `input.key`.

For the name I guess it should be `w3cEventOrNull`

Reply:
w3cEventOrNull is very unfortunate and misleading name, we can name it domEventOrNull which won't be a lie and is more intuitive then webEvent

as of the platform-specific name - what exactly it should be? 
for other platforms we have:
```
androidx.compose.ui.awt.awtEventOrNull
androidx.compose.ui.uikit.uikitEventOrNull
```

Reply:
The key event here is `org.w3c.dom.events.KeyboardEvent` so "w3c" or "dom" should be fine. So, if your vote is for "dom", then
```kt
androidx.compose.ui.dom.domEventOrNull
```

cc @igordmn

Reply:
> as of the platform-specific name - what exactly it should be?

The name of the package/platform of the event class, not the name of the Compose target (for example, `desktop` is a target, but we use `swing`/`javafx`).

So, `dom` sounds good.